docs/deployment/config_descriptions.md

```diff
@@ -90,36 +90,115 @@ For each service, a TaskCountLowAlarm alarm is added to check the RUNNING Task C
               statistic: SampleCount
               threshold: '{{ecs.services.EcsTaskHelloWorld.count}}'
 
-### AutoScaling
+### Cluster AutoScaling
 
 Cumulus ECS clusters have the ability to scale out and in based on
 [CPU and memory reservations](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/cloudwatch-metrics.html#cluster_reservation).
 There are a few configuration values that affect how the ECS cluster instances
 scale:
 
-* `ecs.autoscaling.scaleInThresholdPercent`: the reservation percentage where,
-  if both CPU and memory are under, the EC2 cluster will be scaled in
-* `ecs.autoscaling.scaleOutThresholdPercent`: the reservation percentage where,
-  if both CPU and memory are under, the EC2 cluster will be scaled out
-* `ecs.autoscaling.scalingAdjustmentPercent`: the percentage to increase or
-  decrease the cluster size by when a scaling action is taken. For more
-  information see the
+* `ecs.clusterAutoscaling.scaleInThresholdPercent`: the reservation percentage
+  where, if both CPU and memory are under, the EC2 cluster will be scaled in
+* `ecs.clusterAutoscaling.scaleInAdjustmentPercent`: the percentage to increase
+  or decrease the number of EC2 instances in the cluster when the "scale in"
+  alarm is triggered. Since this is a "scale in" setting, it should typically be
+  a negative value. For more information see the
+  [PercentChangeInCapacity documentation](https://docs.aws.amazon.com/autoscaling/ec2/userguide/as-scaling-simple-step.html#as-scaling-adjustment),
+  specifically the section on `PercentChangeInCapacity`.
+* `ecs.clusterAutoscaling.scaleOutThresholdPercent`: the reservation percentage
+  where, if both CPU and memory are under, the EC2 cluster will be scaled out
+* `ecs.clusterAutoscaling.scaleOutAdjustmentPercent`: the percentage to increase
```

Comment:
I think it would be a good idea to mention here that if this value isn't set, then scaling out will be disabled.